README.md

```diff
@@ -204,7 +207,7 @@ In order to manually test your SDK changes locally, you must use [yarn link](htt
 ```shell
 $ cd packages/firebase
 $ yarn link # initialize the linking to the other folder
-$ cd ../packages/<my-product> # Example: $ cd packages/database
+$ cd ../<my-product> # Example: $ cd ../firestore
```

Comment:
I have to admit I've never used the `yarn link` method of building/linking to an app. I think was correct to begin with, no? The firestore implementation is within `packages/firestore` in our repo.

Reply:
The first step brings you into `packages/firebase`, then this step says you should `cd ../packages/<product>`, which would end up being `packages/packages/<product>`. So it's just a small hiccup that I noticed when I was going through the steps myself.

Reply:
Oh I see!